src/awx_plugins/credentials/plugins.py

```diff
@@ -19,7 +20,7 @@
 )
 
 
-__all__ = ()  # noqa: WPS410
+__all__ = ('hcp_terraform',)  # noqa: WPS410
```

Comment:
We shouldn't really make these public. The tuple is intentionally empty so that there's no public imports exposed.